homeassistant/components/remember_the_milk.py

```diff
@@ -0,0 +1,267 @@
+"""Component to interact with Remember The Milk.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/remember_the_milk/
+
+Minimum viable product, it currently only support creating new tasks in your
+Remember The Milk (https://www.rememberthemilk.com/) account.
+
+Ideas for improvement:
+- add interface to resolve tasks
+- add option to add a task to a certain list
+- add option to list tasks stored in RTM
+- make registration dialog nicer
+"""
```

Comment:
Please keep the file header minimal. The improvements will happen when they happen.

Reply:
ok